spopt/locate/coverage.py

```diff
@@ -61,6 +61,51 @@ def from_cost_matrix(
         Returns
         -------
         LSCP object
+
+        Examples
+        --------
+        >>> from spopt.locate.coverage import LSCP
+        >>> from spopt.locate.util import simulated_geo_points
+        >>> import pulp
+        >>> import spaghetti
```

Comment:
If we are going to have `spaghetti` in the doc string examples, then we should probably also have in `requirements.txt`.